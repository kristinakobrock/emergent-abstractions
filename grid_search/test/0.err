  0%|                                                                                                     | 0/50 [00:00<?, ?it/s]  4%|###7                                                                                         | 2/50 [00:00<00:02, 19.50it/s] 10%|#########3                                                                                   | 5/50 [00:00<00:01, 25.61it/s] 16%|##############8                                                                              | 8/50 [00:00<00:01, 25.51it/s] 24%|######################                                                                      | 12/50 [00:00<00:01, 26.19it/s] 30%|###########################6                                                                | 15/50 [00:00<00:01, 26.07it/s] 36%|#################################1                                                          | 18/50 [00:00<00:01, 25.26it/s] 42%|######################################6                                                     | 21/50 [00:00<00:01, 23.41it/s] 52%|###############################################8                                            | 26/50 [00:01<00:00, 26.91it/s] 58%|#####################################################3                                      | 29/50 [00:01<00:00, 26.55it/s] 64%|##########################################################8                                 | 32/50 [00:01<00:00, 27.36it/s] 70%|################################################################3                           | 35/50 [00:01<00:00, 27.98it/s] 76%|#####################################################################9                      | 38/50 [00:01<00:00, 25.08it/s] 82%|###########################################################################4                | 41/50 [00:01<00:00, 25.24it/s] 88%|################################################################################9           | 44/50 [00:01<00:00, 26.41it/s] 94%|######################################################################################4     | 47/50 [00:01<00:00, 25.04it/s]100%|############################################################################################| 50/50 [00:01<00:00, 25.22it/s]100%|############################################################################################| 50/50 [00:01<00:00, 25.67it/s]
  0%|                                                                                                     | 0/13 [00:00<?, ?it/s] 15%|##############3                                                                              | 2/13 [00:00<00:00, 20.00it/s] 38%|###################################7                                                         | 5/13 [00:00<00:00, 25.63it/s] 62%|#########################################################2                                   | 8/13 [00:00<00:00, 25.74it/s] 85%|#############################################################################8              | 11/13 [00:00<00:00, 27.34it/s]100%|############################################################################################| 13/13 [00:00<00:00, 26.71it/s]