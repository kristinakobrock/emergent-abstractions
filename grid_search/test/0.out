# ["--attributes=3", "--values=4", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=60", "--batch_size=32", "--learning_rate=0.0005", "--hidden_size=128", "--temp_update=0.97", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.6677887439727783, "acc": 0.6191489696502686, "length": 3.551517963409424, "mode": "train", "epoch": 1}
{"loss": 0.6624821424484253, "acc": 0.6251736879348755, "length": 4.0, "mode": "test", "epoch": 1}
{"loss": 0.6572825908660889, "acc": 0.6294456124305725, "length": 3.551011800765991, "mode": "train", "epoch": 2}
{"loss": 0.6535974144935608, "acc": 0.6247016191482544, "length": 3.9166667461395264, "mode": "test", "epoch": 2}
{"loss": 0.651090681552887, "acc": 0.631736159324646, "length": 3.5351903438568115, "mode": "train", "epoch": 3}
{"loss": 0.6371003985404968, "acc": 0.6449652910232544, "length": 4.0, "mode": "test", "epoch": 3}
{"loss": 0.6265350580215454, "acc": 0.6503419280052185, "length": 3.736804485321045, "mode": "train", "epoch": 4}
{"loss": 0.6012226939201355, "acc": 0.6928820610046387, "length": 4.0, "mode": "test", "epoch": 4}
{"loss": 0.5797444581985474, "acc": 0.7066179513931274, "length": 3.917116165161133, "mode": "train", "epoch": 5}
{"loss": 0.5798534154891968, "acc": 0.7008680105209351, "length": 4.0, "mode": "test", "epoch": 5}
{"loss": 0.5528116822242737, "acc": 0.7219001054763794, "length": 3.954242706298828, "mode": "train", "epoch": 6}
{"loss": 0.5789684057235718, "acc": 0.7015625834465027, "length": 4.0, "mode": "test", "epoch": 6}
{"loss": 0.5388246178627014, "acc": 0.7295960187911987, "length": 3.9668095111846924, "mode": "train", "epoch": 7}
{"loss": 0.5790719985961914, "acc": 0.7079861164093018, "length": 4.0, "mode": "test", "epoch": 7}
{"loss": 0.531524121761322, "acc": 0.7335183620452881, "length": 3.9763407707214355, "mode": "train", "epoch": 8}
{"loss": 0.5712178349494934, "acc": 0.7157986164093018, "length": 4.0, "mode": "test", "epoch": 8}
{"loss": 0.5252390503883362, "acc": 0.7331116199493408, "length": 3.984041452407837, "mode": "train", "epoch": 9}
{"loss": 0.5796788930892944, "acc": 0.707812488079071, "length": 4.0, "mode": "test", "epoch": 9}
{"loss": 0.5189514756202698, "acc": 0.7354427576065063, "length": 3.9865686893463135, "mode": "train", "epoch": 10}
{"loss": 0.562253475189209, "acc": 0.7182291150093079, "length": 4.0, "mode": "test", "epoch": 10}
{"loss": 0.5115812420845032, "acc": 0.7405504584312439, "length": 3.9904541969299316, "mode": "train", "epoch": 11}
{"loss": 0.5696026086807251, "acc": 0.7079861164093018, "length": 4.0, "mode": "test", "epoch": 11}
{"loss": 0.5069260001182556, "acc": 0.7425903677940369, "length": 3.9912421703338623, "mode": "train", "epoch": 12}
{"loss": 0.5537633299827576, "acc": 0.7248263955116272, "length": 4.0, "mode": "test", "epoch": 12}
{"loss": 0.5043711066246033, "acc": 0.740455687046051, "length": 3.994364023208618, "mode": "train", "epoch": 13}
{"loss": 0.5457121133804321, "acc": 0.7263888120651245, "length": 4.0, "mode": "test", "epoch": 13}
{"loss": 0.49991562962532043, "acc": 0.7456833720207214, "length": 3.996370792388916, "mode": "train", "epoch": 14}
{"loss": 0.5518066883087158, "acc": 0.7286458015441895, "length": 4.0, "mode": "test", "epoch": 14}
{"loss": 0.49629461765289307, "acc": 0.7471972703933716, "length": 3.996091842651367, "mode": "train", "epoch": 15}
{"loss": 0.5451924800872803, "acc": 0.7269096970558167, "length": 4.0, "mode": "test", "epoch": 15}
{"loss": 0.49150410294532776, "acc": 0.749057412147522, "length": 3.997830390930176, "mode": "train", "epoch": 16}
{"loss": 0.5414007902145386, "acc": 0.7350695729255676, "length": 4.0, "mode": "test", "epoch": 16}
{"loss": 0.4886074662208557, "acc": 0.7514798045158386, "length": 3.9972498416900635, "mode": "train", "epoch": 17}
{"loss": 0.5448752641677856, "acc": 0.7347221970558167, "length": 4.0, "mode": "test", "epoch": 17}
{"loss": 0.48698773980140686, "acc": 0.7526590824127197, "length": 3.998612642288208, "mode": "train", "epoch": 18}
{"loss": 0.5378826856613159, "acc": 0.7373263835906982, "length": 4.0, "mode": "test", "epoch": 18}
{"loss": 0.4801512360572815, "acc": 0.7566553950309753, "length": 3.9978928565979004, "mode": "train", "epoch": 19}
{"loss": 0.5341612100601196, "acc": 0.7432292103767395, "length": 4.0, "mode": "test", "epoch": 19}
{"loss": 0.4775705337524414, "acc": 0.7566035985946655, "length": 3.9989495277404785, "mode": "train", "epoch": 20}
{"loss": 0.5312382578849792, "acc": 0.7388889789581299, "length": 4.0, "mode": "test", "epoch": 20}
{"loss": 0.47066420316696167, "acc": 0.7615363001823425, "length": 3.9990103244781494, "mode": "train", "epoch": 21}
{"loss": 0.5334545969963074, "acc": 0.7333332300186157, "length": 4.0, "mode": "test", "epoch": 21}
{"loss": 0.46554049849510193, "acc": 0.7675115466117859, "length": 3.999382257461548, "mode": "train", "epoch": 22}
{"loss": 0.5214523077011108, "acc": 0.7395833134651184, "length": 4.0, "mode": "test", "epoch": 22}
{"loss": 0.45827028155326843, "acc": 0.775628387928009, "length": 3.9993274211883545, "mode": "train", "epoch": 23}
{"loss": 0.5201348066329956, "acc": 0.7463541030883789, "length": 4.0, "mode": "test", "epoch": 23}
{"loss": 0.4529523253440857, "acc": 0.7781254053115845, "length": 3.999347448348999, "mode": "train", "epoch": 24}
{"loss": 0.5198900699615479, "acc": 0.738020658493042, "length": 4.0, "mode": "test", "epoch": 24}
{"loss": 0.4451500177383423, "acc": 0.7860718965530396, "length": 3.9992799758911133, "mode": "train", "epoch": 25}
{"loss": 0.49745580554008484, "acc": 0.7552083134651184, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.44014713168144226, "acc": 0.7889108061790466, "length": 3.9993598461151123, "mode": "train", "epoch": 26}
{"loss": 0.5159658789634705, "acc": 0.7517361640930176, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.44101032614707947, "acc": 0.7869879603385925, "length": 3.9988064765930176, "mode": "train", "epoch": 27}
{"loss": 0.515189528465271, "acc": 0.7595486044883728, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.4348453879356384, "acc": 0.7935195565223694, "length": 3.99741268157959, "mode": "train", "epoch": 28}
{"loss": 0.4922138750553131, "acc": 0.7630207538604736, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.42572304606437683, "acc": 0.7984799146652222, "length": 3.9991350173950195, "mode": "train", "epoch": 29}
{"loss": 0.49770796298980713, "acc": 0.7562500834465027, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.4218803644180298, "acc": 0.802541196346283, "length": 3.9993717670440674, "mode": "train", "epoch": 30}
{"loss": 0.48076221346855164, "acc": 0.7699652910232544, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.4151643216609955, "acc": 0.8080506920814514, "length": 3.9994559288024902, "mode": "train", "epoch": 31}
{"loss": 0.4908187687397003, "acc": 0.760937511920929, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.4173767566680908, "acc": 0.8024320602416992, "length": 3.9996354579925537, "mode": "train", "epoch": 32}
{"loss": 0.48848050832748413, "acc": 0.766493022441864, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.40827783942222595, "acc": 0.8100318312644958, "length": 3.9997944831848145, "mode": "train", "epoch": 33}
{"loss": 0.47346168756484985, "acc": 0.7715277075767517, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.4052450954914093, "acc": 0.8122866153717041, "length": 3.999122381210327, "mode": "train", "epoch": 34}
{"loss": 0.47824689745903015, "acc": 0.7776041030883789, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.40165552496910095, "acc": 0.8129324316978455, "length": 3.999840497970581, "mode": "train", "epoch": 35}
{"loss": 0.4723891019821167, "acc": 0.7788195013999939, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.39532846212387085, "acc": 0.8169353008270264, "length": 3.999854326248169, "mode": "train", "epoch": 36}
{"loss": 0.4613405466079712, "acc": 0.7805554270744324, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.39338093996047974, "acc": 0.81975919008255, "length": 3.9990155696868896, "mode": "train", "epoch": 37}
{"loss": 0.46904265880584717, "acc": 0.7795138955116272, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.39585304260253906, "acc": 0.8201635479927063, "length": 3.9991772174835205, "mode": "train", "epoch": 38}
{"loss": 0.46003401279449463, "acc": 0.7855902910232544, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.3860764801502228, "acc": 0.8266763091087341, "length": 3.999546766281128, "mode": "train", "epoch": 39}
{"loss": 0.4571751356124878, "acc": 0.7852430939674377, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.3823494613170624, "acc": 0.8291167616844177, "length": 3.999936819076538, "mode": "train", "epoch": 40}
{"loss": 0.4623259902000427, "acc": 0.7875000834465027, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.38304847478866577, "acc": 0.8295790553092957, "length": 3.9997854232788086, "mode": "train", "epoch": 41}
{"loss": 0.4465956687927246, "acc": 0.7994791269302368, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.37649741768836975, "acc": 0.8338040113449097, "length": 3.999650716781616, "mode": "train", "epoch": 42}
{"loss": 0.4424951374530792, "acc": 0.7909722924232483, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.3669852018356323, "acc": 0.8380448222160339, "length": 3.9995718002319336, "mode": "train", "epoch": 43}
{"loss": 0.4527823030948639, "acc": 0.7940971851348877, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.3665531277656555, "acc": 0.8437320590019226, "length": 3.9999070167541504, "mode": "train", "epoch": 44}
{"loss": 0.4535858929157257, "acc": 0.7942708730697632, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.3605075478553772, "acc": 0.841291606426239, "length": 3.999656915664673, "mode": "train", "epoch": 45}
{"loss": 0.4332907497882843, "acc": 0.7993054389953613, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.3585062325000763, "acc": 0.8448936939239502, "length": 3.9999563694000244, "mode": "train", "epoch": 46}
{"loss": 0.42984941601753235, "acc": 0.8109375834465027, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.3497685194015503, "acc": 0.8497093319892883, "length": 3.9999823570251465, "mode": "train", "epoch": 47}
{"loss": 0.445516437292099, "acc": 0.801562488079071, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.3523485064506531, "acc": 0.8470977544784546, "length": 3.9999189376831055, "mode": "train", "epoch": 48}
{"loss": 0.43747758865356445, "acc": 0.8019096851348877, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.3452451825141907, "acc": 0.8526510000228882, "length": 3.998706817626953, "mode": "train", "epoch": 49}
{"loss": 0.43304190039634705, "acc": 0.8118056058883667, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.34030401706695557, "acc": 0.8563215136528015, "length": 3.9999773502349854, "mode": "train", "epoch": 50}
{"loss": 0.4366651773452759, "acc": 0.8010416030883789, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.3400160074234009, "acc": 0.8558403849601746, "length": 3.9996323585510254, "mode": "train", "epoch": 51}
{"loss": 0.4308309555053711, "acc": 0.8092014789581299, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.33639082312583923, "acc": 0.8603820204734802, "length": 3.9999775886535645, "mode": "train", "epoch": 52}
{"loss": 0.4323045313358307, "acc": 0.8027776479721069, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.3393518030643463, "acc": 0.8552204370498657, "length": 3.999995470046997, "mode": "train", "epoch": 53}
{"loss": 0.43746981024742126, "acc": 0.8048611879348755, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.33057716488838196, "acc": 0.8603241443634033, "length": 3.999983549118042, "mode": "train", "epoch": 54}
{"loss": 0.43034929037094116, "acc": 0.8126736879348755, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.32836201786994934, "acc": 0.862929105758667, "length": 3.99828839302063, "mode": "train", "epoch": 55}
{"loss": 0.42332446575164795, "acc": 0.8109375834465027, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.3277778625488281, "acc": 0.8632236123085022, "length": 3.999974012374878, "mode": "train", "epoch": 56}
{"loss": 0.419786661863327, "acc": 0.8206596970558167, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.3289297819137573, "acc": 0.8622968792915344, "length": 3.999992609024048, "mode": "train", "epoch": 57}
{"loss": 0.4304541349411011, "acc": 0.8118056058883667, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.32599276304244995, "acc": 0.8630461692810059, "length": 3.999918222427368, "mode": "train", "epoch": 58}
{"loss": 0.42039549350738525, "acc": 0.8168402910232544, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.3184225261211395, "acc": 0.8683855533599854, "length": 3.9999520778656006, "mode": "train", "epoch": 59}
{"loss": 0.4226684272289276, "acc": 0.823437511920929, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.31725361943244934, "acc": 0.8704167604446411, "length": 3.9999868869781494, "mode": "train", "epoch": 60}
{"loss": 0.4186623692512512, "acc": 0.8244790434837341, "length": 4.0, "mode": "test", "epoch": 60}
test accuracy: 0.7729310989379883
